From dc300721309c7ea8845ae455846c60bcb3b6c3ce Mon Sep 17 00:00:00 2001
Message-Id: <dc300721309c7ea8845ae455846c60bcb3b6c3ce.1676985350.git.git@grubix.eu>
From: David Bremner <david@tethera.net>
Date: Tue, 21 Feb 2023 07:49:15 -0400
Subject: [PATCH] emacs/tree: use two argument form of setq-local

Apparently the macro setq-local only takes two arguments in Emacs 26.1
---
 emacs/notmuch-tree.el | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/emacs/notmuch-tree.el b/emacs/notmuch-tree.el
index 14775d59..b58fa6a6 100644
--- a/emacs/notmuch-tree.el
+++ b/emacs/notmuch-tree.el
@@ -1451,11 +1451,11 @@ notmuch-tree buffers, just set
   (unless (derived-mode-p 'notmuch-tree-mode)
     (user-error "notmuch-tree-outline-mode is only meaningful for notmuch trees!"))
   (if notmuch-tree-outline-mode
-      (progn (setq-local outline-regexp "^[^\n]+"
-			 outline-level #'notmuch-tree-outline--level)
+      (progn (setq-local outline-regexp "^[^\n]+")
+	     (setq-local outline-level #'notmuch-tree-outline--level)
 	     (notmuch-tree-outline--set-visibility))
-    (setq-local outline-regexp (default-value 'outline-regexp)
-		outline-level (default-value 'outline-level))))
+    (setq-local outline-regexp (default-value 'outline-regexp))
+    (setq-local	outline-level (default-value 'outline-level))))
 
 ;;; _
 
-- 
2.39.2.674.gfa067e1834

